jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,465 @@
+= Installing the CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+version:
+- Cloud
+
+NOTE: CircleCI runner is currently only available if you are on the Scale Plan. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+
+This page describes how to install CircleCI runner on Linux and macOS platforms.
+
+toc::[]
+
+=== Prerequisites
+
+==== Support Levels
+
+There are currently three support levels for CircleCI Runner:
+
+* Supported
+* Untested
+* Untested
```

Comment:
Unsupported?

Reply:
fixed

Reply:
"Untested" is intentional - we may or may not support this via their DCE